content/docs/platform/inbox/react/components/preferences.mdx

```diff
@@ -48,3 +48,65 @@ Global subscriber preferences are shown by default. To hide them, add the follow
   display: none;
 }
 ```
+
+### Grouping preferences
+
+Preference grouping lets you organize extensive lists of workflows into well-organized, meaningful groups. This ensures that your end-users get a better experience when managing their preferences.
+ 
+<Callout type="info">
+  Note: Grouping preference is supported in client-side SDKs starting from version 3.4.0.
+</Callout>
+
+To implement preference grouping, use the `preferenceGroups` prop on the `Inbox` component. Each group requires:
+
+- `name`: The text displayed for the group
+- `filter`: Criteria for which preferences appear in the group (object or function)
+
+Here are the key ways to filter preferences:
+
+- **Filter by tags**:
+   ```tsx
+   const groupConfig = { name: 'General', filter: { tags: ['account'] } };
+   ```
+
+- **Filter by tags and workflow IDs**:
+   ```tsx
+   const groupConfig = { name: 'Marketing', filter: { tags: ['marketing'], workflowIds: ['workflow-id'] } };
+   ```
+
+- **Include all workflows** (even those in other groups):
+   ```tsx
+   const groupConfig = { name: 'All Workflows', filter: ({ preferences }) => preferences };
+   ```
```

Comment:
@Aviatorscode2 let's maybe make those snippets not a single line? But prettier them? Because right now they are not the best to read. 

Also, can we maybe consider a different display for them and not in bullets (not sure I like how it looks like(